managed/ui/src/components/config/Storage/StorageConfigurationContainer.js

```diff
@@ -26,6 +27,18 @@ const mapDispatchToProps = (dispatch) => {
   return {
     addCustomerConfig: (config) => {
       return dispatch(addCustomerConfig(config)).then((response) => {
+        if (response.error) {
+          const errorMessageObject = response.payload?.response?.data?.error || response.payload.message;
+          Object.keys(errorMessageObject).forEach((errorKey) => {
```

Comment:
Since all the other instance of error payload is different than this I have not done this, we can create a generic component to handle all categories.

Reply:
For that, we must have some definite cases